src/v1/data_transfer_service_client.ts

```diff
@@ -145,13 +145,6 @@ export class DataTransferServiceClient {
     // identifiers to uniquely identify resources within the API.
     // Create useful helper objects for these.
     this._pathTemplates = {
-      projectDataSourcePathTemplate: new gaxModule.PathTemplate(
-        'projects/{project}/dataSources/{data_source}'
-      ),
-      projectLocationDataSourcePathTemplate: new gaxModule.PathTemplate(
-        'projects/{project}/locations/{location}/dataSources/{data_source}'
-      ),
-      projectPathTemplate: new gaxModule.PathTemplate('projects/{project}'),
```

Comment:
@xiaozhenliu-gg5 We lost this common resource :(

Reply:
Seems `projectLocationDataSourcePathTemplate` and `projectDataSourcePathTemplate ` are added  by the end of the pathTemplate List, so the only missing resource is `project`. And I checked the proto file, this resource is not referenced so not rendered in the list.